docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
   - /topics/scripting/guide.md
 ---
 
-TODO: update per v2
-
 # Scripting Guide
 
-This guide explains the basics of how to create and structure `brigade.js` files.
+This guide explains the basics of how to create and structure Brigade scripts,
+which can be written in either JavaScript (`brigade.js`) or TypeScript
+(`brigade.ts`).
 
 # Brigade Scripts, Projects, and Repositories
 
-Brigade scripts are stored in a `brigade.js` file. They are designed to contain short scripts
-that coordinate running multiple related tasks. We like to think of these as
-cluster-oriented shell scripts: The script just ties together a bunch of other programs.
+Brigade scripts can be stored either in a project definition or in a project's
+git repository. They are designed to organize and coordinate running multiple
+related tasks, usually in response to events. We like to think of these as
```

Comment:
Nit: What do you think about dropping "usually?" There really is no case where the script _isn't_ triggered by an event.